src/main/java/org/folio/dao/rollover/LedgerFiscalYearRolloverDAO.java

```diff
@@ -46,4 +45,20 @@ public Future<Void> delete(String id, DBClient client) {
     return promise.future();
   }
 
+  public Future<Boolean> validationOfUniqueness(String sql, DBClient client) {
```

Comment:
I couldn't decipher the intent of this method easily. Something seems off about a method that takes any sql statement and does "validation of uniqueness" that lives on DAO class for rollover and it is public. A name/interface change should be in order here. Does this method validate the uniqueness of the result of the query? Should this method be on the `LedgerFiscalYearRolloverDAO` class or some utility class?

Reply:
the name of this method should start with a verb. ex. `validateUniqueness`.

Reply:
Fixed. Method renamed to isRolloverExists and moved to RolloverValidationService